packages/web-components/src/components/va-button/va-button.scss

```diff
@@ -17,65 +15,78 @@
 
 // Override transparent background from USWDS v3
 .usa-button--outline {
-  background-color: var(--color-white);
+  background-color: $color-white;
 }
 
-/* Original Component Styles. */
-
-@import '../../mixins/buttons.css';
-
-:host(:not([uswds])) button {
-  align-items: center;
-  background-color: var(--color-primary);
-  border-radius: 5px;
-  color: var(--color-white);
-  display: flex;
-  font-size: 1.6rem;
-  justify-content: center;
-  margin: 0.5em 0.5em 0.5em 0;
-}
-
-:host(:not([uswds])) button:hover,
-:host(:not([uswds])) button:focus {
-  background-color: var(--color-primary-darker);
+:host(:not([uswds])) {
+  button {
+    align-items: center;
+    appearance: none;
+    background-color: $color_primary;
+    border-radius: 5px;
+    border: 0;
+    color: $color-white;
+    cursor: pointer;
+    display: flex;
+    font-family: $font-family-sans;
+    font-size: v1_rem_to_px(1.6rem);
+    font-weight: 700;
+    justify-content: center;
+    line-height: 1;
+    margin: 0.5em 0.5em 0.5em 0;
```

Comment:
`em` doesn't need to be adjusted?

Reply:
Had to google it because I wasn't sure why:

> The `rem` unit is only relative to the document's root element, while the `em` unit is only relative to the immediate parent of the targeted element.